app/views/proposals_answers_mailer/notify_massive_import.html.erb

```diff
@@ -0,0 +1,4 @@
+<p class="email-greeting"><%= I18n.t("decidim.events.email_event.email_greeting", user_name: @user.name) %></p>
+<p class="email-instructions"><%= I18n.t("decidim.events.proposals.proposals_answers_publisheds_for_space.email_intro", participatory_space_title: @participatory_space_title).html_safe %></p>
```

Comment:
review the publisheds word please

Reply:
Replacing key like previous suggestion. Thanks!